packages/zowe-explorer/i18n/sample/src/job/actions.i18n.json

```diff
@@ -21,5 +21,7 @@
   "cancelJobs.notImplemented": "The cancel function is not implemented in this API.",
   "cancelJobs.notCancelled": "The job was not cancelled.",
   "cancelJobs.failed": "One or more jobs failed to cancel: {0}",
-  "cancelJobs.succeeded": "Cancelled selected jobs successfully."
+  "cancelJobs.succeeded": "Cancelled selected jobs successfully.",
+  "filterJobs.message": "Inorder to filter jobs,first populate them using search icon",
```

Comment:
Could we change this message to say the same as if profile was expanded without search in place.
`Use the search button to display jobs` 
Or something along those lines, possibly `A jobs search must be in place before local filtering is possible`